DuckDB.NET.Data/DuckDBSchema.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Data;
+using System.Linq;
+using System.Text;
+
+namespace DuckDB.NET.Data;
+
+internal static class DuckDBSchema
+{
+    public static DataTable GetSchema(DuckDBConnection connection, string collectionName, string?[]? restrictionValues)
+    {
+        return collectionName.ToUpperInvariant() switch
+        {
+            "METADATACOLLECTIONS" => GetMetaDataCollections(),
```

Comment:
_System.Data.Common.DbMetaDataCollectionNames_ defines schema constants.

Reply:
Thanks for the feedback. I know, but unfortunately `DbMetaDataCollectionNames` has collection names as PascalCase, but the collection names themselves need to be case insensitive. It would be more code with no benefit, and potentially slower.

Reply:
In other words, I would not be able to use switch expression here, as it requires a constant in the left part.

Reply:
Sure, but the compiler converts the switch expression into if/else expressions. In this way you can use the constants defined by Ado.Net.

Reply:
I know that it would be converted to if statements by the compiler. But here I care more about readability and expressiveness of the code. Trying to rewrite it into the if-else would make the code uglier.